reference/configuration/framework.rst

```diff
@@ -1852,6 +1852,8 @@ started on every session initialization. The probability is calculated by
 using ``gc_probability`` / ``gc_divisor``, e.g. 1/100 means there is a 1%
 chance that the GC process will start on each request.
 
+If not set, ``php.ini``'s `session.gc_probability`_ directive will be relied on.
```

Comment:
Maybe with a versionadded directive to indicate this is a new behavior ? 
And we also need to remove default value few lines above

Reply:
Thanks for the feedback ! Just committed your recommandations.